### ISSUE TITLE
Comments: consistent commenting,  unused code and Radspec

### ISSUE THREAD

**Issue by Super-Nim** [Encode-Solidity-Bootcamp-Group-10/Week1#3]
There are intermittent comments in the script files. it's best to comment EVERYTHING to drill the purpose of each constant, object, provider, etc. accomplishes into our minds. Although it may seem tedious, repetition is key to success 🔑  

Some files have NO comments at all - let this be a thing of the past.

_Intermittent commenting:_

<img width="589" alt="Screenshot 2022-07-09 at 12 12 33" src="https://user-images.githubusercontent.com/64858288/178103199-5c26dbec-a317-4a1f-8d83-94043b483838.png">

Also, l've noticed unused/commented out code left behind. Let's keep it clean and avoid this when possible.

_unused code:_
<img width="575" alt="Screenshot 2022-07-09 at 12 12 51" src="https://user-images.githubusercontent.com/64858288/178103218-eb83d86f-bfe7-4430-92db-7ad87917dc01.png">

We haven't been commenting on our smart contracts at all. I also propose that we implement [Radspec](https://github.com/aragon/radspec) in our Smart Contracts from now on too.

**Comment by danisev7** [comment]
I totally agree to improve the documentation. We should open some pull requests to update the documentation to the [NatSpec Format](https://docs.soliditylang.org/en/latest/natspec-format.html#natspec) and [Radspec](https://github.com/aragon/radspec)
